docs/documentation/release_notes/topics/26_4_0.adoc

```diff
@@ -1,42 +1,56 @@
 // Release notes should contain only headline-worthy new features,
 // assuming that people who migrate will read the upgrading guide anyway.
 
-Read on to learn more about each new feature, and https://www.keycloak.org/docs/latest/upgrading/index.html[find additional details in the upgrading guide] if you are upgrading from a previous release of {project_name}.
+This release features new capabilities focused on security enhancements, deeper integration, and improved server administration. The highlights of this release are:
 
-= Federated client authentication preview
+* Passkeys for seamless, passwordless authentication of users
+* Federated Client Authentication to use SPIFFE or Kubernetes service account tokens for client authentication
+* Simplified deployments across multiple availability zones to boost availability
+* Serving as an authorization server when connecting AI application via the Model Context Protocol (MCP)
 
-Identity providers are now able to federate client authentication. This allows clients to authenticate with SPIFFE JWT SVIDs,
-Kubernetes service accounts, or tokens issued by a OpenID Connect identity provider.
+Read on to learn more about each new feature, and https://www.keycloak.org/docs/latest/upgrading/index.html[find additional details in the upgrading guide] if you are upgrading from a previous release of {project_name}.
```

Comment:
This seems to indicate that the upgrade guide contains more details on new features and enhancements, which it doesn't.